client/post-editor/term-token-field/index.jsx

```diff
@@ -96,8 +99,13 @@ TermTokenField.propTypes = {
 export default connect( ( state, props ) => {
 	const siteId = getSelectedSiteId( state );
 
+	const postId = getEditorPostId( state );
+	const postType = getEditedPostValue( state, siteId, postId, 'type' );
+	const taxonomy = getPostTypeTaxonomy( state, siteId, postType, props.taxonomyName );
+
 	return {
-		siteId: siteId,
+		siteId,
+		taxonomyLabel: taxonomy && taxonomy.label || props.taxonomyName,
```

Comment:
Without parentheses, it's not obvious to me the intended behavior here. Is it to always check for presence of `taxonomy` and use `label`, falling back to `props.taxonomyName`? Or is it to fallback to `props.taxonomyName` when either `taxonomy` or `taxonomy.label` are falsey?

Aside: Some folks use [`_.get`](https://lodash.com/docs#get) for these kinds of deep path lookups, e.g.:

```js
taxonomyLabel: get( taxonomy, 'label', props.taxonomyName )
```